archaius2-test/src/main/java/com/netflix/archaius/test/Archaius2TestConfig.java

```diff
@@ -35,255 +35,238 @@
  * 
  * See {@link TestPropertyOverride} for additional usage information.
  */
-public class Archaius2TestConfig implements TestRule, Config {
-    private final ThreadLocal<Config> configHolder = new ThreadLocal<>();
+public class Archaius2TestConfig implements TestRule, Config, SettableConfig {
```

Comment:
Consider omitting the explicit mention of the Config interface, given that SettableConfig extends Config.